doc/sphinx/proofs/writing-proofs/inductive.rst

```diff
@@ -1110,13 +1108,15 @@ symbol :g:`=`.
 
 .. tacv:: simplify_eq
 
-   If the current goal has form :g:`t1 <> t2`, it behaves as
+   If the current goal has form :g:`t1 <> t2`, it behaves like
    :n:`intro @ident; simplify_eq @ident`.
 
-.. tacn:: dependent rewrite -> @ident
+.. tacn:: dependent rewrite {? {| -> | <- } } @one_term {? in @ident }
    :name: dependent rewrite ->
```

Comment:
Should this be in the "Reasoning with equalities" chapter?

Reply:
Yes, that could make sense. But a pointer to this tactic should appear in this chapter given the note that this kind of equalities may be derived by the `injection` and `inversion` tactics.

Reply:
Ah, on the other hand, this is already part of the "Equality and inductive sets" section which is entirely on the fringe between the two chapters.

Reply:
Your final answer is "keep it in this chapter", but (I expect) add a hyperlink to it in the equality chapter?

Reply:
No, I don't have a final answer. But if you move it, then add a link.